### PR TITLE
Update network request guide to include info about disabling logging

### PR DESCRIPTION
**Pull request by mjhenkes**

### CODE REVIEW

File: docs/guides/guides/network-requests.mdx [hunk 1]
```diff
@@ -541,6 +542,17 @@ test in the Command Log. Here is an example of what this looks like:
   alt="Screenshot of fetch and XHR requests"
 />
 
+:::info
+
+**Note:** Logging can be disabled off by passing `{ log: false }` with your
```
Comment (mschile) [inline docs/guides/guides/network-requests.mdx:547]:
```suggestion
**Note:** Logging can be disabled by passing `{ log: false }` with the
```

File: docs/guides/guides/network-requests.mdx [hunk 2]
```diff
@@ -541,6 +542,17 @@ test in the Command Log. Here is an example of what this looks like:
...
+static response.
+
+```js
+cy.intercept('/users*', { body: ['user1', 'user2'], log: false }).as('getUsers')
```
Comment (mschile) [inline docs/guides/guides/network-requests.mdx:551]: Should we use the example that disables all xhr/fetch logs?

https://docs.cypress.io/api/commands/intercept#Disabling-logs-for-a-request

Comment (mjhenkes) [reply docs/guides/guides/network-requests.mdx:551]: I specifically used the example from farther down in the code to match: https://github.com/cypress-io/cypress-documentation/pull/5124/files#diff-bb8ce895b7660f9f324902a3e98c85d05f942fdd16997394dd5fc0dc46abb095L554

but could change to disable all if people want.

Comment (mschile) [reply docs/guides/guides/network-requests.mdx:551]: Could also link out to the example that shows how to disable all if we didn't want to update here.

Comment (mjhenkes) [reply docs/guides/guides/network-requests.mdx:551]: updated in c020484

File: docs/guides/guides/network-requests.mdx [hunk 3]
```diff
@@ -525,8 +525,9 @@ cy.wait('@new-user').then(console.log)
 
 ## Command Log
 
-Cypress logs all `XMLHttpRequest`s and `fetch`es made by the application under
-test in the Command Log. Here is an example of what this looks like:
+By default Cypress logs all `XMLHttpRequest`s and `fetch`es made by the
```
Comment (AtofStryker) [inline docs/guides/guides/network-requests.mdx:528]:
```suggestion
By default, Cypress logs all `XMLHttpRequest`s and `fetch`es made by the
```

**Review by AtofStryker — APPROVED** [review]

**Pull request merged**